starport/pkg/chaincmd/runner/chain.go

```diff
@@ -16,12 +16,12 @@ import (
 
 // Start starts the blockchain.
 func (r Runner) Start(ctx context.Context, args ...string) error {
-	return r.run(ctx, runOptions{longRunning: true}, r.cc.StartCommand(args...))
+	return r.run(ctx, runOptions{wrapStdErr: false}, r.cc.StartCommand(args...))
```

Comment:
This is a possible memory leak tho, I'm not sure that it will be noticiable. Stargate for some reason outputs standard logs from stderror also, so stderr streams tends to get bigger than usual.

It would be ideal to use a custom buffer to only save a limited KBs of latest logs in memory instead of saving from the beginning.

~ not blocking for me.

Reply:
I'm checking if it can be solved

Reply:
Thanks for reporting it. The outputs are quite extensive, so I preferred to develop a simple solution to truncate the logs instead of shipping as then.
Here we simply truncate the log when it reaches a specific threshold.

It keeps only the first bytes but this is fine because we want to parse start errors and so the log to parse will always be from the first bytes.